website/util/sanitize.py

```diff
@@ -58,3 +58,17 @@ def _ensure_clean(value):
             raise ValueError
 
     return escape_html(data)
+
+
+def unescape_html(s):
```

Comment:
A more descriptive name would be helpful, since this doesn't completely unescape html; it only unescapes escape sequences that we deem safe. Maybe `safe_unescape_html`?